mne/surface.py

```diff
@@ -855,6 +863,35 @@ def _call_old(self, rr, n_jobs):
         return inside
 
 
+class _CheckInsideSphere:
+    def __init__(self, sphere):
+        from .bem import ConductorModel
+
+        assert isinstance(sphere, ConductorModel) and sphere["is_sphere"]
+        self.cm = sphere["r0"]
+        self.inner_r = sphere.radius  # float or None
```

Comment:
```suggestion
        self.center = sphere["r0"]
        self.radius = sphere.radius  # float or None
```

Reply:
So this needs to match `_CheckInside`, which works on a surface. It has `cm` for "center of mass", `inner_r` for "inner radius" or min distance from center of mass to any point, and `outer_r` or "outer radius" for max distance from center of mass to any point. I could at least change `.cm` to `.center`, but `.inner_r` probably has to stay as is for compat with the surface case. (For a sphere, `inner_r==outer_r` essentially.)